Sources/ElasticSwift/Admin/Indices.swift

```diff
@@ -16,32 +16,19 @@ public class IndiciesAdmin {
         self.client = withClient
     }
     
-    public func createIndex() -> CreateIndexRequestBuilder {
-        return CreateIndexRequestBuilder(withClient: self.client)
+    func createIndex(withName name: String) -> CreateIndexRequestBuilder {
```

Comment:
There are some mandatory fields needed to perform ES operations.
I.e. the index name cannot be NIL on createIndex operation.
So it's better to make it non optional in the RequestBuilders and adapt all constructors to require it.